src/deploy/deployTestStratSingleAsset2.ts

```diff
@@ -6,13 +6,12 @@ const deploy: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { deploy } = deployments
   const { deployer } = await getNamedAccounts()
   const BalleMaster = await deployments.get('BalleMaster')
-  const TokenA = await deployments.get('TokenA')
   const TokenB = await deployments.get('TokenB')
 
   await deploy('TestStratSingleAsset2Token', {
     contract: 'TestStrategy',
     from: deployer,
-    args: [BalleMaster.address, TokenA.address, TokenB.address],
+    args: [BalleMaster.address, TokenB.address],
```

Comment:
Just to learn about it, why do we keep Token 'B' instead of 'A'?

Reply:
We have two test single asset strategies, one with TKNA and another with TNKB